packages/synthchron-tauri/src/components/react-flow/processModels/petriNet/petriNetFlowConfig.ts

```diff
@@ -60,7 +60,8 @@ export const petriNetFlowConfig: ProcessModelFlowConfig = {
       },
       data: {
         label: node.name,
-        store: isPlaceNode(node) ? node.amountOfTokens : undefined,
+        store: isPlaceNode(node) ? node.amountOfTokens : node.weight,
+        accepting: (node as any).accepting,
```

Comment:
Shouldn't be _any_, I think we should define types for the "data" field of Places and Transitions, but don't need to do that now. Creating a ticket